synapse/storage/databases/main/room.py

```diff
@@ -1191,6 +1198,60 @@ def _background_add_rooms_room_version_column_txn(txn: LoggingTransaction):
 
         return batch_size
 
+    async def _background_populate_rooms_creator_column(
+        self, progress: dict, batch_size: int
+    ):
+        """Background update to go and add creator information to `rooms`
+        table from `current_state_events` table.
+        """
```

Comment:
Add tests for this background update

ex.

 - `test_background_update_multiple_rooms`
 - `test_background_update_single_large_room`